### ISSUE TITLE
startRequestTimer is unavailable in production mode but not mentioned in the docs

### ISSUE THREAD

**Issue by zhyupe** [balderdashy/sails#2807]
Refering to: https://github.com/balderdashy/sails/blob/master/lib/hooks/http/middleware/defaults.js#L70
```javascript
startRequestTimer: !IS_PRODUCTION && function startRequestTimer(req, res, next) {
  req._startTime = new Date();
  next();
},
```
In production mode, this middleware isn't work, and ```req._startTime``` is undefined. But this feature is not mentioned in the docs.
When using ```req._startTime``` without determination, it would cause unexcepted confusing error.

**Comment by tjwebb** [comment]
What version of sails are you running?

**Comment by zhyupe** [comment]
Latest version (0.11.0)

The docs says "the pre-installed stack is perfectly acceptable for most apps in development or production". But in fact, something in it behaves differently in development and production mode. (The timer, etc.)

**Comment by luislobo** [comment]
Yes, I fixed it adding the following to /config/http.js
```
 startRequestTimer: function startRequestTimer(req, res, next) {
      req._startTime = new Date();
      next();
    },
```

**Comment by sailsbot** [comment]
Thanks for posting, @zhyupe.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
